packages/core/admin/server/controllers/permission.js

```diff
@@ -29,11 +29,9 @@ module.exports = {
    * @param {KoaContext} ctx - koa context
    */
   async getAll(ctx) {
-    const { role: roleId } = ctx.query;
+    const { sectionsBuilder, actionProvider, conditionProvider } = getService('permission');
 
-    const { sectionsBuilder, conditionProvider } = getService('permission');
-
-    const actions = await getService('action').getAllowedActionsForRole(roleId);
```

Comment:
Even though we don't need to handle an edge case for the author role, it might be a good idea to keep validating that the given role is a valid one (it exists in the DB). What do you think?

Reply:
Note: We don't send the role anymore